[DA-2607] MIGRATION Add diversion pouch to genomics

brianmkimmel opened the pull request:
## Resolves *[DA-2607](https://precisionmedicineinitiative.atlassian.net/browse/DA-2607)*


## Description of changes/additions

Add diversion pouch flag to GenomicSetMember.
Update W1IL query to check for diversion pouch flag.
Remove checks for existance of files that are checked when GenomicSetMember.qcStatus is set.

## Tests
- [] unit tests

brianmkimmel committed:
Add diversionPouchSiteFlag to GenomicSetMember
Modify W1IL query to remove file checks and add check for diversionPouchSiteFlag
Update unit tests to test that row is not included if diversionPouchSiteFlag is set and remove test for craiFile not found.

brianmkimmel committed:
Add diversionPouchSiteFlag to GenomicSetMember
Modify W1IL query to remove file checks and add check for diversionPouchSiteFlag
Update unit tests to test that row is not included if diversionPouchSiteFlag is set and remove test for craiFile not found.

brianmkimmel committed:
Merge branch 'devel' into bk/DA-2607

dtharpeuno commented on rdr_service/dao/genomics_dao.py:
```diff
@@ -3417,19 +3401,9 @@ def get_data_ready_for_w1il_manifest(self, module: str, cvl_id: str):
                 ParticipantSummary.consentForGenomicsROR == QuestionnaireStatus.SUBMITTED,
                 GenomicGCValidationMetrics.drcFpConcordance.ilike('pass'),
-                sample_collected_site.siteType != 'diversion pouch',
+                GenomicSetMember.diversionPouchSiteFlag == 0,
```
Can we update this to `!=1`, as this is the convention we use in the rest of the queries.

brianmkimmel replied:
Yes, I've committed that change.

brianmkimmel committed:
Update comparison to match query convention.
Add diversionPouchFlag check to new W1IL query.
Add imports back for query order_by

brianmkimmel committed:
Remove line accidentally duplicated in merge
